### ISSUE TITLE
mitochondrial DNA

### ISSUE THREAD

**Issue by pontikos** [vplagnol/pipelines#20]
We should map reads to MT DNA too.
Is it in the reference?

**Comment by vplagnol** [comment]
yes definitely in the reference. I just don't have all the merged gVCF files for these, only some. But yes we should include this as well.

**Comment by pontikos** [comment]
Ok, I guess if we have DNA samples from mitochondrial-rich cell types that
would be good resource.

**Comment by vplagnol** [comment]
Mostly matters for Shamima Rahman who studies mitochondrial diseases
